pkgs/applications/audio/sonixd/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, fetchurl
+, appimageTools
+}:
+
+let
+  pname = "sonixd";
+  version = "0.14.0";
+  name = "sonixd-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/jeffvli/sonixd/releases/download/v${version}/Sonixd-${version}-linux-x86_64.AppImage";
+    sha256 = "sha256-q+26Ut5wN9gFDBdqirR+he/ppu/b1wiqq23WkcRAQd4=";
+  };
+
+in
+appimageTools.wrapType2 {
```

```suggestion
appimageTools.wrapType2 rec {
```
with this we could move everything down and remove the `let...in`

---

rec is a little bit a matter of style. I prefer without rec, so that you know exactly which variables are being defined.